core/profiles/standard/standard.install

```diff
@@ -364,6 +367,23 @@ function standard_install() {
     ->fields(array('uid' => 1, 'role' => $admin_role->name))
     ->execute();
 
+  // Create a default editor role for users who need to be able to edit content but do not need to access
+  // advanced admin features.
+  $admin_role = new stdClass();
```

Comment:
All these variables should be `$editor_role` instead:

```suggestion
  $editor_role = new stdClass();
```

Reply:
I did that and it didn't work. I broke the site. Will try again to be sure.

Reply:
I am trying again to see if it was user error the first time.

Reply:
This time it's working. I must have done something wrong the first time.